aws/templates/createProductTemplate.ftl

```diff
@@ -17,7 +17,7 @@
     [#case "productInHost"]
     [#default]
         [#assign productDomain = productDomainStem]
-        [#assign productDomainQualifier = "-" + productName]
+        [#assign productDomainQualifier = "-" + productDomain]
```

Comment:
I don't think this change should be made. Because the switch assumes you want to include the product in the host portion of the domain, it should be the productName not the productDomain that is included